src/coreComponents/unitTests/virtualElementTests/testVirtualElementBase.cpp

```diff
@@ -40,7 +40,7 @@ static void checkIntegralMeanConsistency( VEMBASISDATATYPE const & basisData )
   {
     sum += basisFunctionsIntegralMean[iBasisFun];
   }
-  EXPECT_TRUE( abs( sum-1 ) < 1e-15 )
+  EXPECT_TRUE( std::abs( sum-1 ) < 1e-15 )
```

Comment:
is `std::abs` available in cuda? I want to say I had always used `fabs`.

Reply:
Use `LvArray::math::abs`, it's the best of both worlds.

Reply:
Done.